src/cpp/session/modules/SessionPackages.R

```diff
@@ -962,29 +962,59 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
    ## Create a DESCRIPTION file
    
    # Fill some bits based on devtools options if they're available.
-   # Protect against vectors with length > 1
-   getDevtoolsOption <- function(optionName, default, collapse = " ")
+   # Protect against vectors with length > 1.
+   getDevtoolsOption <- function(optionName,
+                                 default = NULL,
+                                 collapse = " ")
    {
-      devtoolsDesc <- getOption("devtools.desc")
-      if (!length(devtoolsDesc))
-         return(default)
-      
-      option <- devtoolsDesc[[optionName]]
-      if (is.null(option))
-         return(default)
+      for (descKey in c("usethis.description", "devtools.desc"))
+      {
+         descValue <- getOption(descKey)
+         if (length(descValue))
+         {
+            optionValue <- descValue[[optionName]]
+            if (length(optionValue))
+            {
+               if (inherits(optionValue, "person"))
+               {
+                  optionValue <- .rs.formatPerson(optionValue)
+               }
+               else if (is.list(optionValue))
+               {
+                  for (i in seq_along(optionValue))
+                  {
+                     if (inherits(optionValue[[i]], "person"))
+                     {
+                        optionValue[[i]] <- .rs.formatPerson(optionValue[[i]])
+                     }
+                  }
+               }
+               
+               return(paste(optionValue, collapse = collapse))
```

Comment:
With the huge caveat that I'm running this in my head, I do wonder ... does this really do the right thing if `usethis.description[["Authors@R"]]` is a list of `person()`s? 🤔 Who's writing the surrounding `c()` and the commas?

I see some logic around that in `.rs.formatPerson` but it feels like we'll never apply that to the whole list of `person()`s but instead to each individually?

Reply:
It should, since a list of persons is still just a "person" object, e.g.

```
> me <- person("Kevin", "Ushey")
> class(c(me, me))
[1] "person"
```